data/859/074/65/85907465.geojson

```diff
@@ -4,21 +4,20 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
-    "geom:area":0.0,
+    "geom:area":0.000112,
     "geom:area_square_m":0.0,
-    "geom:bbox":"46.719684,24.649513,46.719684,24.649513",
-    "geom:latitude":24.649513,
-    "geom:longitude":46.719684,
+    "geom:bbox":"46.714982,24.640226,46.733836,24.651113",
+    "geom:latitude":24.64588,
+    "geom:longitude":46.722714,
     "gn:id":"",
     "iso:country":"SA",
-    "lbl:bbox":"46.714684,24.644513,46.724684,24.654513",
-    "lbl:latitude":24.649513,
-    "lbl:longitude":46.719684,
+    "lbl:latitude":24.64567,
+    "lbl:longitude":46.721143,
     "lbl:max_zoom":18.0,
-    "mps:latitude":24.649191,
-    "mps:longitude":46.727386,
-    "mz:hierarchy_label":0,
-    "mz:is_current":-1,
+    "mps:latitude":24.64567,
+    "mps:longitude":46.721143,
+    "mz:hierarchy_label":1,
```

Comment:
So all the new neighbourhood geoms are flipping the bit from 0 to 1 for including in PIP results string label construction? That makes sense now they are good polygons.

Reply:
Yep, any neighbourhood record with a new polygon will now appear in the label string (hierarchy label = 1).